package.json

```diff
@@ -127,7 +129,9 @@
     "serve": "gatsby serve",
     "type-check": "tsc --noEmit",
     "storybook": "storybook dev -p 6006 -c .storybook",
-    "build-storybook": "storybook build"
+    "build-storybook": "storybook build",
+    "theme": "chakra-cli tokens src/@chakra-ui/gatsby-plugin/theme.ts",
```

Comment:
Nice. Interesting decision to place those generated types in node_modules and not on the project 🤔.